src/build/Workleap.DotNet.CodingStandards.props

```diff
@@ -29,4 +29,20 @@
     <!-- GenerateDocumentationFile must be set to true for IDE0005 (Remove unnecessary usings/imports) to work -->
     <GenerateDocumentationFile Condition="'$(GenerateDocumentationFile)' == ''">true</GenerateDocumentationFile>
   </PropertyGroup>
+
+   <PropertyGroup>
+        <!-- Enable NuGet package auditing -->
```

Comment:
Could we add this to the comments for us to remember where it does come from / how to configure it: https://learn.microsoft.com/en-us/nuget/concepts/auditing-packages